### ISSUE TITLE
add administrative command destroyBlock, destroyFrontier

### ISSUE THREAD

**Issue by paulvilla** [StunlockStudios/vrising-dedicated-server-instructions#49]
As an administrator I have already found myself in the situation of having to deal with two groups, in which one of them intentionally built a whole barrier to prevent the other from entering its base.

As an administrator, I believe that it is necessary to be able to control these abusive and uncompetitive techniques of the community.

**Comment by Jofrais** [comment]
oh please yes

**Comment by misterzieta** [comment]
Yes we need this so much! I banned some users doing this but not sure if their castle will disappear.

**Comment by paulvilla** [comment]
No, even if you beat the person, their constructions will remain on the server until the blood runs out in the castle.
In the last update a command was added that allows to break almost everything except the most important, expansion plots and where the blood is stored, with which the user can connect after deleting almost everything and lock again:

Keep in mind that there are people who do certain things to screw or expel people without the need to play, this causes many people to leave the servers and others may even leave the game because of these techniques.

I think it is important to be able to manage these buildings for the good of the server and also of the game

**Comment by misterzieta** [comment]
paulvilla thanks for the info and imput! what is the command you are talking about?

**Comment by paulvilla** [comment]
> paulvilla thanks for the info and imput! what is the command you are talking about?

the best option would be that when a person has the admin command active and interacts with the "hearts of the castle" they see it the same as the owner.
So if you have to destroy a castle it would simply remove the blood and could hit the demolish button.

With this implementation it would not be necessary to add new commands

**Comment by paulvilla** [comment]
Look, this is an example of what is happening.

I don't know if it's a bug or people breaking into other people's servers, with hacks.
We recently did a wipe to remove these things and in the first 8 hours I already have similar problems

You have to give the administrators tools to be able to destroy these things

![image](https://user-images.githubusercontent.com/14887229/171289932-7c98c33d-e2c7-4adc-b641-028a22c36c54.png)

**Comment by misterzieta** [comment]
Dear devs, any news on this basic request?

**Comment by codecisco** [comment]
good afternoon, in the game there is absolutely no command to delete someone else's building in administrator mode?
